schemas/when_rule/compare_to_list_count.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "type": "object",
+  "description": "Comparison against a list count",
+  "properties": {
+    "list": {
+      "type": "string",
```

Comment:
Personally, I think it is beneficial to check that the list specified here does actually exist within the schema?

Reply:
I've added a check for this in the latest commit